roles/tackle/templates/serviceaccount-ui.yml.j2

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ ui_serviceaccount_name }}
+  namespace: {{ app_namespace }}
+  annotations:
+    serviceaccounts.openshift.io/oauth-redirectreference.primary: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"{{ ui_route_name }}"}}'
```

Comment:
Normally I would argue OLM should create an SA, but it needs this annotation for the oauth-proxy to work, so the template pretty much needs to exist in full, even to add the annotation...

Reply:
Perhaps that's the way there , we let OLM manage the service account itself for the sake of consistency (and cleanliness) , then patch it when openshift-oauth is needed/selected by user ?

Reply:
I have no problem with that. Right now it'll just be created by the template as tackle-ui, mta-ui or whatever based on profile. It'd probably have to be updated to mta-ui for downstream in the CSV? If that isn't a problem I'm OK with adding it to the CSV. As far as I know it requires no special permissions so it'd be one line.